weed/command/filer_sync.go

```diff
@@ -65,6 +67,8 @@ func init() {
 	syncOptions.bProxyByFiler = cmdFilerSynchronize.Flag.Bool("b.filerProxy", false, "read and write file chunks by filer B instead of volume servers")
 	syncOptions.aDebug = cmdFilerSynchronize.Flag.Bool("a.debug", false, "debug mode to print out filer A received files")
 	syncOptions.bDebug = cmdFilerSynchronize.Flag.Bool("b.debug", false, "debug mode to print out filer B received files")
+	syncOptions.aSyncFromTs = cmdFilerSynchronize.Flag.Int("a.syncFromTs", 0, "synchronization from timestamp on filer A. The unit is millisecond")
```

Comment:
How about just use fromTs instead of syncFromTs?

Reply:
I agree with you.
It's easier to understand with the unit.